app/client/models/AuditLogsModel.ts

```diff
@@ -45,7 +45,7 @@ export class AuditLogsModel extends Disposable implements AuditLogsModel {
         return;
       }
 
-      this.streamingDestinations.set(value);
+      this.streamingDestinations.set(value as AuditLogStreamingDestinations);
```

Comment:
I'm not sure why this and the `Config.ts` was changed, you are not using it anywhere, right? I think it can be introduced later on with some actual usage.

Reply:
I've reworked the code and now there's an actual use for this. It's used for adding the latest version to the current home server in case it doesn't have it yet. This is important for the case of multiple home servers.